azure-pipelines/vcpkg-arm64/Dockerfile

```diff
@@ -0,0 +1,59 @@
+# This infrastructure is adapted from that the .NET team uses, see
+# https://github.com/dotnet/dotnet-buildtools-prereqs-docker
+# and
+# https://github.com/dotnet/runtime/blob/main/docs/workflow/building/coreclr/linux-instructions.md
+FROM mcr.microsoft.com/cbl-mariner/base/core:2.0
+
+RUN tdnf install -y \
+        # Common utilities
+        ca-certificates \
+        git \
+        tar \
+        curl \
+        # Build tools
+        binutils \
+        cmake \
+        ninja-build \
+        # Rootfs build dependencies
+        glibc-devel \
```

Comment:
This won't do anything because we aren't using the glibc from cbl-mariner in the first place, we're using the one from Ubuntu xenial

Reply:
libc6-devel then? https://packages.ubuntu.com/search?searchon=contents&keywords=scrt1.o&mode=&suite=oracular&arch=any